.github/workflows/monitor-studios.yml

```diff
@@ -0,0 +1,28 @@
+name: Test Public Studios
+on:
+  schedule:
+    cron: '0 0 12 * * ?' # Fire at 12:00 p.m. (noon) every day
+
+jobs:
+  test-and-build:
```

Comment:
small thing: may `e2e-studio-cron-test ` or anything you prefer

Reply:
Good point. Changed it now.